tests/helper/helper.go

```diff
@@ -151,6 +173,26 @@ func WaitForDeploymentReplicaCount(t *testing.T, kc *kubernetes.Clientset, name,
 	return false
 }
 
+// Waits until the number of ready replicas hits target or number of iterations are done.
+func WaitForDeploymentReadyReplicaCount(t *testing.T, kc *kubernetes.Clientset, name, namespace string,
```

Comment:
Instead of creating a new function, I think that there's no harm to update `WaitForDeploymentReplicaCount()` to wait on Ready as well, there shouldn't be any problem.
@v-shenoy @JorTurFer WDYT?

Reply:
What do you mean ready 'as well'?

Reply:
@v-shenoy the change from:
```go
replicas := deployment.Status.Replicas
```
to 
```go
replicas := deployment.Status.ReadyReplicas
```

Reply:
Yeah, sorry for the confusion :)